### DISCUSSION TITLE
Bring the UI menu to the center of the screen like windows 11 taskbar icons for ultrawide monitors.

### DISCUSSION THREAD

**Discussion by Ice2192** [OpenRCT2/OpenRCT2#21988] [Feature requests]
Windows 11 has a feature to bring the taskbar icons from the left corner of the screen to the middle which helps a lot for people with ultrawide monitors. It's a feature that I wish could make it's way to OpenRCT2 if it's possible for players with ultrawide monitors so that its not a head shake distance.

![328306652-64d473f7-0027-46fb-a6b4-3a05abfca772](https://github.com/OpenRCT2/OpenRCT2/assets/35847819/d292caad-1205-4eaf-ae58-3b64d8dd587c)

**Comment by AaronVanGeffen** [comment]
This is a great idea. It's been brought up before, but no-one's ever implemented it. See https://github.com/OpenRCT2/OpenRCT2/discussions/15643 for some more discussion.

Closing this as a duplicate.
